packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -52,6 +52,11 @@ class PinCodeBloc extends $PinCodeBloc {
     required this.autoPromptBiometric,
   }) {
     authenticated.connect().addTo(_compositeSubscription);
+    Rx.merge([
+      _$addDigitEvent.switchMap((digit) => _addDigit(digit).asStream()),
+      _$deleteDigitEvent
+          .map((_) => _pinCode.value.substring(0, _pinCode.value.length - 1)),
+    ]).listen((event) => _pinCode.value = event).addTo(_compositeSubscription);
```

Comment:
Can we use method reference to improve the readability ?

Reply:
Yes, done